src/initialize.cpp

```diff
@@ -134,6 +134,9 @@ parse_command_line(int argc, char* argv[])
       } else if (arg == "-n" || arg == "--particles") {
         i += 1;
         settings::n_particles = std::stoll(argv[i]);
+      
+      } else if (arg == "-e" || arg == "--event") {
```

Comment:
Can you also add this flag to `man/man1/openmc.1` and `docs/source/usersguide/scripts.rst`?